source/administration/services/hdbpp/hdbpp-es-interface.rst

```diff
@@ -0,0 +1,272 @@
+
+
+
+Device properties
+~~~~~~~~~~~~~~~~~
+
++-----------------------------+------------------------------------------------------------------+
+| AttributeList               | list of configured attributes                                    |
++-----------------------------+------------------------------------------------------------------+
+| CheckPeriodicTimeoutDelay   | delay before timeout when checking periodic events, in seconds   |
++-----------------------------+------------------------------------------------------------------+
+| PollingThreadPeriod         | default period for polling thread, in seconds                    |
++-----------------------------+------------------------------------------------------------------+
+| LibConfiguration            | configuration parameters for backend support library             |
++-----------------------------+------------------------------------------------------------------+
+| ContextsList                | definition of possible archiver operating contexts               |
++-----------------------------+------------------------------------------------------------------+
+| DefaultStrategy             | default context for a new attribute (Default to RUN)             |
++-----------------------------+------------------------------------------------------------------+
+| StatisticsTimeWindow        | timeslot for statistics                                          |
++-----------------------------+------------------------------------------------------------------+
+| SubscribeRetryPeriod        | retry period for subscribe event, in seconds                     |
++-----------------------------+------------------------------------------------------------------+
+
+Table 5: Event Subscriber Device properties.
+
+In addition to the already described Class properties, device Properties
+comprehend the AttributeList property which contains the list of
+attributes in charge of the current device. The syntax is
+*fully-qualified-attribute-name;context=CONTEXT* where *CONTEXT* can be
+one or a combination of the defined contexts (logic OR). Whenever not
+specified the DefaultContext specified in the Class property or in the
+Device Property applies. Table shows some examples:
+
+.. code-block:: console
+   :linenos:
+
+   $ tango://srv-tango-srf.fcs.elettra.trieste.it:20000/eos/climate/18b20 eos.01/State;context=RUN|SHUTDOWN
+   $ tango://srv-tango-srf.fcs.elettra.trieste.it:20000/eos/climate/18b20 eos.01/Temperature;context=RUN|SHUTDOWN
+   $ tango://srv-tango-srf.fcs.elettra.trieste.it:20000/ctf/diagnostics/ccd_ctf.01/State;context=RUN
+   $ tango://srv-tango-srf.fcs.elettra.trieste.it:20000/ctf/diagnostics/ccd_ctf.01/HorProfile;context=RUN
+   $ tango://srv-tango-srf.fcs.elettra.trieste.it:20000/ctf/diagnostics/ccd_ctf.01/VerProfile;context=RUN
+
+Table 6: AttributeList example
+
+The first two attributes will be archived in both RUN and SHUTDOWN
+contexts; the last three only when in RUN.
+
```

Comment:
Following is a general description of the Configuration manager. Suggest to move at the beginning of hdbpp-cm-interface.rst

Reply:
Another option i to move it in the general section...